lib/smart_answer_flows/check-uk-visa/outcomes/outcome_standard_visitor_visa.erb

```diff
@@ -1,12 +1,10 @@
-<% text_for :title do %>
-  You’ll need a visa to come to the UK
-<% end %>
-
 <% govspeak_for :body do %>
+  ## You’ll need a visa to come to the UK
+
   Apply for a [Standard Visitor visa](/standard-visitor-visa).
 
   <% if calculator.passport_country_is_china? %>
-    ##You're travelling in a tour group
+    ## You're travelling in a tour group
```

Comment:
I wonder if this should be an h3 as it looks like a sub-clause to the main section.